.editorconfig

```diff
@@ -568,3 +568,5 @@ dotnet_diagnostic.SA1402.severity = none
 dotnet_diagnostic.SA1401.severity = none
 ### Private fields should not start with underscore
 dotnet_diagnostic.SA1309.severity = none
+### Use "gets" for properties docstrings
+dotnet_diagnostic.SA1623.severity = none
```

Comment:
### Author's Note

With all the properties for the parameter objects being added, this was going to get really annoying if it wasn't disabled.